### DISCUSSION TITLE
Quality assessment

### DISCUSSION THREAD

**Discussion by Visioneery** [bloc97/Anime4K#191] [Q&A]
Hi!
Is there any information about how was the assessment of image quality done except visual inspection of images?
Did you use any no reference methods and pre trained models  to determine that upscaled image has a better quality than original?

Best regards

**Comment by bloc97** [comment]
The restore shader was assessed to be better than nothing using a single-blind experiment involving a dozen people. The upscale shaders were tested using PSNR against bilinear and other upscalers. There are also third party tests that you can find online done by other people.

**Reply by Visioneery** [reply]
Hi! 
I tried to use Anime4k upscaling GLSL algorithms and as I can understand it applying a shader on it , it obviously changes the size of image, but the amount of pixels stays the same.
However upscaling  typically refers to the process of increasing resolution or increasing the amount of pixels in image while preserving its quality and details. 
So another question would be : how  did you prove that image is upscaled using Anime4k algorithms?
